### PR TITLE
Addressing Issue #10

### PR DESCRIPTION
**Pull request by Redorhcs**
Closes issue #10  from the Yalies dev side:

- Updating the User model in models.py to have the same properties as in Yalies (https://github.com/Yalies/api/blob/master/app/models.py#L15-L22). **(Change made in models.py)**
- Making sure that all references to now-renamed properties (such as username which will now be called id) elsewhere in the codebase are renamed accordingly. **Only one change found in routes.py**

Requires change from @ErikBoesen to handle the DB deletion once this is verified

### CODE REVIEW

File: app/routes.py [hunk 1]
```diff
@@ -12,21 +12,21 @@
...
 def store_user():
     if request.method != 'OPTIONS':
-        if cas.username:
-            g.user = User.query.get(cas.username)
+        if cas.id:
+            g.user = User.query.get(cas.id)
             timestamp = int(time.time())
             if not g.user:
-                g.user = User(username=cas.username,
+                g.user = User(id=cas.id,
```
Comment (ErikBoesen) [inline app/routes.py:19]: The CAS package will still use the name `username`.

File: app/routes.py [hunk 2]
```diff
@@ -12,21 +12,21 @@
...
                               registered_on=timestamp)
                 db.session.add(g.user)
             g.user.last_seen = timestamp
             db.session.commit()
-            print('NetID: ' + cas.username)
+            print('NetID: ' + cas.id)
```
Comment (ErikBoesen) [inline app/routes.py:24]: Same here

File: app/routes.py [hunk 3]
```diff
@@ -12,21 +12,21 @@
...
 
 
 @app.route('/')
 def index():
-    if not cas.username:
+    if not cas.id:
```
Comment (ErikBoesen) [inline app/routes.py:29]: And here

**Review by ErikBoesen — CHANGES_REQUESTED** [review]
Just a quick fix - otherwise this looks excellent, thank you!

File: app/routes.py (hunk 3 above)
Comment (Redorhcs) [reply app/routes.py:29]: Thanks for catching that!

**Review by ErikBoesen — APPROVED** [review]

**Pull request merged**